### PR TITLE
a11y: improve date color contrast

### LINKED ISSUE #68: a11y: sufficient contrast
Elements such as timestamps in the list of posts are very difficult to see: they have extremely low contrast. Their size is also set in absolute units (px) instead of relative units (em, rem, `%`), so they might not have their size increased by a "minimum-size" browser preference; this makes their low contrast a much more significant issue, as small elements need higher contrast to be visible.

I'd recommend using [a contrast calculator](https://www.myndex.com/APCA/) to pick a foreground color with better perceptual contrast.

### PR DESCRIPTION
**Pull request by Seirdy**
Closes #68

### CODE REVIEW

File: _sass/general.scss [hunk 1]
```diff
@@ -16,6 +16,7 @@ body {
     font-size: $size-font-normal;
     line-height: 1.75em;
     color: $color-text-normal;
+		background-color: $color-white;
```
Comment (jwillmer) [inline _sass/general.scss:19]: Why did you add `background-color`? I don't see any usage of it.

**Review by jwillmer — CHANGES_REQUESTED** [review]

File: _sass/general.scss (hunk 1 above)
Comment (Seirdy) [reply _sass/general.scss:19]: When users set default browser colors to e.g. white-on-black and the website sets a dark custom foreground color, the background also needs adjustment. Otherwise you'll have dark text on a dark background.

Comment (Seirdy) [reply _sass/general.scss:19]: Wrote a bit more about this issue in the "Default colors" section of https://seirdy.one/posts/2020/11/23/website-best-practices/#default-colors

**Review by jwillmer — APPROVED** [review]
